packit_service/worker/checker/abstract.py

```diff
@@ -35,5 +38,6 @@ def _pre_check(self) -> bool:
 
     def pre_check(self) -> bool:
         if not self.actor:
-            return False
+            logger.debug("Actor not set for this event, skipping the actor check.")
+            return True
         return self._pre_check()
```

Comment:
Aren't we checking also other non-actor conditions? (I went through the code really quickly and am not sure about the `pre_check` / `_pre_check` call chain...)

Reply:
e.g. project allowed for internal TF,..

Reply:
I think only `_pre_check`s in subclasses [here](https://github.com/packit/packit-service/blob/d126fe4839f54c7b4ab7e6bd4232d2e709930208/packit_service/worker/checker/copr.py#L90) or [here](https://github.com/packit/packit-service/blob/d126fe4839f54c7b4ab7e6bd4232d2e709930208/packit_service/worker/checker/testing_farm.py#L63) are called, but I would wait for Maja's approval since she knows the code the best

Reply:
Thanks Laura for fixing this. 
I did not realize I had changed the actor checker behaviour. You are right, before my changes it would have returned `False` only with a valid `actor` field.

Before my changes this code were called for every handler, but most of them just returned `True`, now it is called just for this two handlers Laura linked here.